yolox/exp/base_exp.py

```diff
@@ -72,6 +72,17 @@ def merge(self, cfg_list):
             if hasattr(self, k):
                 src_value = getattr(self, k)
                 src_type = type(src_value)
+
+                # pre-process input if source type is a sequence
+                if isinstance(src_value, Sequence):
```

Comment:
IMHO, `Sequence` is not suitable here. String type is also seqence, 
```python
isintance(src_value, (tuple, list))
```
might be better. WDYT?